lib/package.json

```diff
@@ -1,12 +1,16 @@
 {
    "name": "node-task-mgr",
-   "version": "1.1.0",
+   "version": "1.1.5",
```

Comment:
TODO: I think it time to change major version because of getProcList->getProcessList.